src/items/static-items.md

```diff
@@ -6,18 +6,17 @@
 >              `=` [_Expression_] `;`
 
 A *static item* is similar to a [constant], except that it represents a precise
-memory location in the program. A static is never "inlined" at the usage site,
-and all references to it refer to the same memory location. Static items have
-the `static` lifetime, which outlives all other lifetimes in a Rust program.
-Static items may be placed in read-only memory if the type is not [interior 
-mutable]. Static items do not call `drop` at the end of the program.
+memory location in the program. All references to the static refer to the same
+memory location. Static items have the `static` lifetime, which outlives all
```

Comment:
Should this be `'static` lifetime instead?

Reply:
`'static` is the `static` lifetime. I think both could work there.